azure-monitor-opentelemetry/samples/tracing/http_fastapi.py

```diff
@@ -26,6 +24,7 @@ async def exception():
     raise Exception("Hit an exception")
 
 
+# Set the OTEL_PYTHON_EXCLUDE_URLS environment variable to "http://127.0.0.1:8000/exclude"
```

Comment:
Should add this env var to the README

Reply:
I guess we could. It's part of the instrumentation's readmes, but we did include it in the docs. There are other instrumentation related env vars I have not delt with much, so I think it would make sense to leave that for another PR where I add a whole section for instrumentation configuration env vars